lib/resolvers.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+class Resolvers {
+  constructor(context) {
+    this.sdk = context.accessors.sdk;
+
+    this._resolversMap = {
+      Query: {
+        allBooks: async () => {
+          const results = await this.sdk.document.search(
+            'test-data',
+            'books'
+          );
+
+          return results.hits.map((hit) => {
+            return { id: hit._id, ...hit._source };
+          });
+        },
+        book: async (root, { id }) => {
+          const result = await this.sdk.document.get(
+            'test-data',
+            'books',
+            id
+          );
+
+          return { id: result._id, ...result._source };
+        },
+        author: async (root, { name }) => {
+          const authorResults = await this.sdk.document.search(
+            'test-data',
+            'authors',
+            {
+              query: {
+                term: {
+                  name: name,
```

Comment:
Nit: you can use shorthand notation
```suggestion
                  name,
```